api/areaProfile.go

```diff
@@ -180,17 +181,16 @@ func (s *Server) placeProfile(c *gin.Context) {
 		resources[i].Name = name
 	}
 
-	sort.SliceStable(resources, func(i, j int) bool {
-		return resources[i].Ratings > resources[j].Ratings // Inverse sort
-	})
 	// metric
 	metric, err := s.mongoStore.SyncAccountPOIMetrics(accountNumber, profile.ScoreCoefficient, poiID)
 	if err != nil {
 		c.Error(err)
 		abortWithEncoding(c, http.StatusInternalServerError, errorInternalServer, err)
 		return
 	}
-	score, _, delta := score.CalculatePOIAutonomyScore(resources, *metric)
+
+	autonomyScore, _, autonomyScoreDelta := score.CalculatePOIAutonomyScore(allResources, *metric)
```

Comment:
Shall we just use the score from db? Otherwise the neighborhood score might be changed due to new reports, and the value is only refreshed here, but it's not reflected on list POI API.